### ISSUE TITLE
pl.Datetime only accessible via pl.DataType.Datetime with "module": "es6" and "type": "module"

### ISSUE THREAD

**Issue by sezanzeb** [pola-rs/nodejs-polars#55]
#### Have you tried latest version of polars?

- [yes]

#### What version of polars are you using?

0.7.3

#### What operating system are you using polars on?

5.15.94-1-MANJARO

#### What node version are you using

node 18

#### Describe your bug.

If the package.json says type: module

```json
{
  "name": "fresh-ts-project",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "type": "module",
  "dependencies": {
    "@types/node": "^18.15.11",
    "nodejs-polars": "^0.7.3",
    "typescript": "^5.0.2"
  }
}
```

tsconfig

```json
{
  "compilerOptions": {
    "module": "es6",
    "moduleResolution": "node"
  },
  "include": [
    "main.ts"
  ]
}
```

and main.ts

```js
import pl from 'nodejs-polars'
console.log('a', pl.Datetime)
// @ts-ignore
console.log('b', pl.DataType.Datetime)
```

it prints

```bash
tsc && node main.js 
a undefined
b [Function: Datetime]
```

I don't remember the specifics why we have those two values in our tsconfig and `"type": "module"`, but it's usually because imports of other packages are broken without it.

furthermore, typescript will think that pl.DataType.Datetime is an error, unless it is imported via `import * as pl from 'nodejs-polars';`.

**Comment by sezanzeb** [comment]
Turns out the project still runs with `"module": "CommonJS",` and without `"type": "module"`. I hope it will stay this way :)

The peculiarities of imports in javascript are still somewhat of a mystery to me.

I don't know if fixing this is important, or if the scenario provided here is instead some weird annoying edge-case configuration that is too troublesome to look at.

**Comment by universalmind303** [comment]
> The peculiarities of imports in javascript are still somewhat of a mystery to me.

Was looking more into this, seems like a JS modules issue 🤷‍♂️. FWIW we mostly test around CommonJS as it's still the officially supported module system, other ones _may_ not work as expected. It'd be nice to set up a test suite to test the 1000 different module systems in JS.

**Comment by Bidek56** [comment]
This works fine in ES6 and Bun: `import { pl } from 'nodejs-polars'`

**Comment by gplanansky** [comment]
FWIW  Deno 1.39.3 imports nodejs-polars fine, and under it polars 0.8.4  wants the `pl.DataType.Datetime` invocation.   Ditto using the deno kernel in jupyter-lab 4.0.10.

Here's a  javascript shot at the opening example in https://docs.pola.rs/user-guide/basics/reading-writing/ :

```
$ deno
Deno 1.39.3
exit using ctrl+d, ctrl+c, or close()
REPL is running with all permissions allowed.
To specify permissions, run `deno repl` with allow flags.
> import pl from "npm:nodejs-polars";
undefined
> let df;
undefined
> df = pl.DataFrame(
    {
        "integer": [1,2,3],
        "date": [
            Date.parse('01 Jan 2022 00:00:00 GMT-0500'),
            Date.parse('02 Jan 2022 00:00:00 GMT-0500'),
            Date.parse('03 Jan 2022 00:00:00 GMT-0500'),
        ],
        "float": [4.0, 5.0, 6.0],
    }
)

Object [DataFrame] {
  integer: shape: (3,)
  Series: 'integer' [f64]
  [
  	1.0
  	2.0
  	3.0
  ],
  date: shape: (3,)
  Series: 'date' [f64]
  [
  	1.6410e12
  	1.6411e12
  	1.6412e12
  ],
  float: shape: (3,)
  Series: 'float' [f64]
  [
  	4.0
  	5.0
  	6.0
  ]
}

> df.withColumns(pl.col("date").cast(pl.DataType.Datetime('ms')))
Object [DataFrame] {
  integer: shape: (3,)
  Series: 'integer' [f64]
  [
  	1.0
  	2.0
  	3.0
  ],
  date: shape: (3,)
  Series: 'date' [datetime[ms]]
  [
  	2022-01-01 05:00:00
  	2022-01-02 05:00:00
  	2022-01-03 05:00:00
  ],
  float: shape: (3,)
  Series: 'float' [f64]
  [
  	4.0
  	5.0
  	6.0
  ]

> df.withColumns(pl.col("date").cast(pl.Datetime('ms')))
Uncaught TypeError: pl.Datetime is not a function
    at <anonymous>:1:60
> df.withColumns(pl.col("date").cast(pl.datetime('ms')))
Uncaught TypeError: pl.datetime is not a function
    at <anonymous>:1:60
> df.withColumns(pl.col("date").cast(pl.dateTime('ms')))
Uncaught TypeError: pl.dateTime is not a function
    at <anonymous>:1:60

```

**Comment by universalmind303** [comment]
this should be resolved in `0.10.0` via https://github.com/pola-rs/nodejs-polars/pull/187. Feel free to reopen the issue if needed.
